python/modules/IcePy/Proxy.cpp

```diff
@@ -1928,7 +1954,7 @@ checkedCastImpl(ProxyObject* p, const string& id, PyObject* facet, PyObject* ctx
         }
 
         AllowThreads allowThreads; // Release Python's global interpreter lock during remote invocations.
-        b = target->ice_isA(id, c);
+        b = target->ice_isA(id, (ctx == 0 || ctx == Py_None) ? Ice::noExplicitContext : c);
```

Comment:
```suggestion
        b = target->ice_isA(id, (ctx == nullptr || ctx == Py_None) ? Ice::noExplicitContext : c);
```

Reply:
Is nullptr preferred for raw pointers too? There is a bunch of these actually!

Reply:
Pretty sure. https://en.cppreference.com/w/cpp/language/nullptr